src/components/Profile/Profile.js

```diff
@@ -10,146 +10,175 @@ import Avatar from '../styleElements/avatar/Avatar.js';
 // import axios from 'axios';
 
 export function Profile(size) {
-	const location = useLocation();
-
-	const likedFilms = [];
-	for (let i = 0; i < 3; i++) {
-		likedFilms.push(dummyData[i]);
-	}
-
-	const topMatches = [];
-	for (let i = 4; i < 7; i++) {
-		topMatches.push(dummyData[i]);
-	}
-
-	const [ posts, setPosts ] = useState([]);
-
-	// useEffect(() => {
-	// 	axios
-	// 		.get('https://jsonplaceholder.typicode.com/posts')
-	// 		.then((res) => {
-	// 			console.log(res);
-	// 			setPosts(res.data);
-	// 		})
-	// 		.catch((err) => {
-	// 			console.log(err);
-	// 		});
-	// }, []);
-
-	// 	useEffect(() => {
-	// 		const response = await axios({
-	// 			method: 'PUT',
-	// 			withCredentials: true,
-	// 			url: `${serverURL}/likeTracker/like`,
-	// 			data: { film: filmArray[0] },
-	// 		});
-	// 		console.log(response.data);
-	// 		return response.data;
-	// 	} catch (err) {
-	// 		return err;
-	// 	}
-	// }
-
-	return (
-		<div className='profile__container'>
-			<div className='profile__header'>
-				<div className='profile__settings'>
-					<Link to='/dashboard/ProfileEdit'>
-						<Settings className={location.pathname === '/dashboard/ProfileEdit' ? 'activeLogo' : null} />
-					</Link>
-				</div>
-
-				<Link to='/'>
-					<p className='profile__logout' active={location.pathname === '/'}>
-						{' '}
-						Logout
-					</p>
-				</Link>
-			</div>
-
-			<div className='profile__avatar-container'>
-				<Avatar />
-			</div>
-
-			{/* <div>
+  const location = useLocation();
+
+  const likedFilms = [];
+  for (let i = 0; i < 3; i++) {
+    likedFilms.push(dummyData[i]);
+  }
+
+  const topMatches = [];
+  for (let i = 4; i < 7; i++) {
+    topMatches.push(dummyData[i]);
+  }
+
+  const [posts, setPosts] = useState([]);
+
+  // useEffect(() => {
+  // 	axios
+  // 		.get('https://jsonplaceholder.typicode.com/posts')
+  // 		.then((res) => {
+  // 			console.log(res);
+  // 			setPosts(res.data);
+  // 		})
+  // 		.catch((err) => {
+  // 			console.log(err);
+  // 		});
+  // }, []);
+
+  // 	useEffect(() => {
+  // 		const response = await axios({
+  // 			method: 'PUT',
+  // 			withCredentials: true,
+  // 			url: `${serverURL}/likeTracker/like`,
+  // 			data: { film: filmArray[0] },
+  // 		});
+  // 		console.log(response.data);
+  // 		return response.data;
+  // 	} catch (err) {
+  // 		return err;
+  // 	}
+  // }
+
+  return (
+    <div className="profile__container">
+      <div className="profile__header">
+        <div className="profile__settings">
+          <Link to="/dashboard/ProfileEdit">
+            <Settings
+              className={
+                location.pathname === '/dashboard/ProfileEdit'
+                  ? 'activeLogo'
+                  : null
+              }
+            />
+          </Link>
+        </div>
+
+        <Link to="/">
+          <p
+            className="profile__logout"
+            active={location.pathname === '/' ? 'true' : 'false'}
```

Comment:
the only thing I actually changed in the Profile.js file was to make this conditional return a string because when it returned a boolean it was causing an error

Reply:
ez